src/API/Reports/PerformanceIndicators/Controller.php

```diff
@@ -167,11 +167,10 @@ public function get_analytics_report_data() {
 						continue;
 					}
 
-					$stat                  = $prefix . '/' . $property_key;
-					$this->allowed_stats[] = $stat;
-					$stat_label            = empty( $schema_info['title'] ) ? $schema_info['description'] : $schema_info['title'];
-
-					$this->labels[ $stat ]  = trim( preg_replace( '/\W+/', ' ', $stat_label ) );
+					$stat                   = $prefix . '/' . $property_key;
+					$this->allowed_stats[]  = $stat;
+					$stat_label             = empty( $schema_info['title'] ) ? $schema_info['description'] : $schema_info['title'];
+					$this->labels[ $stat ]  = trim( preg_replace( '/\.+/', ' ', $stat_label ) );
```

Comment:
Is the intention to remove leading or trailing periods? If so, I think we can use the second argument to `trim()` instead: `$this->labels[ $stat ]  = trim( $stat_label, '.' );`

Reply:
So I think we had a standardization of labels at one point, thinking @LevinMedia led the charge there. But based on  @psealock's comment [here](https://github.com/woocommerce/woocommerce-admin/issues/4854#issuecomment-709593078) I do think all we are trying to do here is replace the trailing period.